json-examples/svc/provider-network-interas-option-a.json

```diff
@@ -0,0 +1,51 @@
+{
+   "attachment-circuits":{
+      "ac":{
+         "name":"IPT-CUST-ABC",
+         "customer-name":"CUST-ABC",
+         "description":"CUST-ABC-113",
+         "peer-sap-id":"sap#113",
+         "ip-connection":{
+            "ipv4":{
+               "local-address":"113.1.1.1",
```

Comment:
```suggestion
               "local-address":"192.0.2.1",
```

Reply:
use documentation address block as defined in [RFC 5737](https://datatracker.ietf.org/doc/html/rfc5737)